content/_data/logo/brno.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: 'Brno'
+url: '/images/logos/brno/brno.png'
+url_256: '/images/logos/brno/brno256.png'
+credit:
```

Comment:
CC @olivergondza who may know whom to credit

Reply:
All credit goes to Maxwell Arbuckle who provided those and who I believe is also the author.
We also have the working formats in case someone is interested: https://github.com/BrnoJAM/jam/tree/master/materials

Reply:
Thanks @olivergondza ! I will add it as a follow-up task